samples/android/GleanCrashExample/build.gradle

```diff
@@ -0,0 +1,30 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = '1.3.50'
+    repositories {
+        google()
+        jcenter()
+        
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.1'
```

Comment:
Let's use `classpath "com.android.tools.build:gradle:$rootProject.versions.android_gradle_plugin"` here so that we don't need two different gradle versions to build the samples for this repo.

Reply:
See 'standalone' comment above